lib/active_merchant/billing/gateways/agms.rb

```diff
@@ -0,0 +1,322 @@
+# Agms Gateway uses nokogiri to build the xml request
+require 'nokogiri'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class AgmsGateway < Gateway
+      self.test_url = 'https://gateway.agms.com/roxapi/agms.asmx'
```

Comment:
Since test_url and live_url are the same, how are test requests distinguished from live requests?